linked_pages.js

```diff
@@ -0,0 +1,24 @@
+document.querySelector(".instruction_button").addEventListener('click', function () {
```

Comment:
this query selector could be stored in a variable so you don't need to query twice

Reply:
gotcha 👍